addons/mrp/models/stock_picking.py

```diff
@@ -106,6 +115,29 @@ def action_detailed_operations(self):
         action['context']['has_kits'] = self.has_kits
         return action
 
+    @api.depends('group_id.stock_move_ids.created_production_id.procurement_group_id.mrp_production_ids',
+                'move_ids.move_dest_ids.raw_material_production_id')
+    def _compute_mrp_production_ids(self):
```

Comment:
Respect the [method convention](https://www.odoo.com/documentation/master/contributing/development/coding_guidelines.html#symbols-and-conventions): compute methods before action methods